### PR TITLE
remove template literals

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: app/views/idb-celebrations/idb-profile.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ define(['app','data/idb-celebrations/links','lodash','directives/idb-celebration
           _ctrl.article = o.data
 
           if(_ctrl.article.id)
-            window.open(`https://www.cbd.int/management/articles/${_ctrl.article.id}/edit`, '_blank')
+            window.open('https://www.cbd.int/management/articles/'+_ctrl.article.id+'/edit', '_blank')
```
Comment (sbilodeau) [inline app/views/idb-celebrations/idb-profile.js]: part of the security policy we should always encode url parameters :(
`encodeURIComponent()`
```
'https://www.cbd.int/management/articles/'+encodeURIComponent(_ctrl.article.id)+'/edit'
```

Comment (sbilodeau) [reply app/views/idb-celebrations/idb-profile.js]: `window.open()` is very bad practice and always get blocked by pop-up blockers. 
Normally it does not work in defered mode! You are lucky if it worksw

If you can it's better to use ```<a target="_blank" ng-href="{{articleUrl}}">...</a>```

Comment (randyhoulahan) [reply app/views/idb-celebrations/idb-profile.js]: articleUrl is obtained from a promise.  Any Idea how to use ng-href when the target url is obtained from a promise?
 ng-href="{{await articleUrl()}}"

Comment (randyhoulahan) [reply app/views/idb-celebrations/idb-profile.js]: Blaise suggested programmatically triggering click(), however, it works in all browsers except safari by asking the user if they permit the popup.  This is for internal use, as a result, I do not think I should spend more time here.  What do you think?

Comment (sbilodeau) [reply app/views/idb-celebrations/idb-profile.js]: I agree. just add the `encodeURIComponent(_ctrl.article.id)`. To comply with security policy

Comment (randyhoulahan) [reply app/views/idb-celebrations/idb-profile.js]: done

**Pull request merged**